apps/deploy-web/src/components/onboarding/OnboardingContainer/OnboardingContainer.spec.tsx

```diff
@@ -135,14 +163,27 @@ describe("OnboardingContainer", () => {
 
   function setup(
     input: {
-      paymentMethods?: any[];
-      user?: any;
+      paymentMethods?: Array<{ id: string; type: string }>;
+      user?: { emailVerified?: boolean; userId?: string };
+      wallet?: { hasManagedWallet?: boolean; isWalletLoading?: boolean };
       windowLocation?: Partial<Location>;
       windowHistory?: Partial<History>;
+      savedStep?: string;
     } = {}
   ) {
     jest.clearAllMocks();
 
+    // Mock localStorage
+    const mockLocalStorage = {
+      getItem: jest.fn().mockReturnValue(input.savedStep || null),
+      setItem: jest.fn(),
+      removeItem: jest.fn()
+    };
+    Object.defineProperty(window, "localStorage", {
+      value: mockLocalStorage,
+      writable: true
+    });
```

Comment:
This is actually the same as using jest global mocks and is a sign of issues with DI. `localStorage` should be a part of the component `DEPENDNECIES` so that it can be hooked up via DI.